.github/workflows/build_and_test.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         working-directory: ./p4sde/tools/setup
         run: |
               sudo apt update -y
-              sudo apt install python python3-setuptools python3-pip python3-wheel python3-cffi sudo libssl-dev git zip libatomic1 libnl-route-3-dev curl tree -y
+              sudo apt install python python3-setuptools python3-pip python3-wheel python3-cffi sudo libssl-dev git zip libatomic1 libnl-route-3-dev curl tree libgtest-dev -y
```

Comment:
We supply Google Test (libgtest) as part of the Stratum dependencies (the same place we get gRPC, Protobuf, etc.). Adding it to the global environment only confuses tings.

Reply:
I added this for testing, will remove in next patch!

Reply:
removed!